src/vos/README.md

```diff
@@ -372,163 +343,45 @@ VOS byte-array objects also support punch over both partial and complete extent
-The additional metadata at each node creates a bounding box representation with {start-offset, end-offset, min-epoch, max-epoch} representing a rectangle.
-Although the representation of each write can be a rectangle, since the tree is ordered by start offset during insertion, two rectangles which are close together in a 2D space can potentially land in the left and right subtrees.
-It would be better to represent this tree as a bounding box based spatial tree capable of traversing rectangles rather than points.
-
 <a id="7j"></a>
 
-<b>Augmented tree to address excessive splitting of nodes while minimizing splitting for inserts in future epochs</b>
-
-![../../doc/graph/Fig_015.png](../../doc/graph/Fig_015.png "Example of four Storage Nodes, eight DAOS Targets and three DAOS Pools")
-
-R-Trees provide a good way to represent dynamically added bounded rectangles.
-In the case of an epoch-extent tree, rectangles might not be bounded initially as required by R-Trees, but overwrites create bounded rectangles with splitting.
-So a variant of an R-Tree that can allow splitting of rectangles dynamically would be required to address this problem.
-
-We propose an Extent - Validity trees (EV-Trees) to address this problem.
-An EV-Tree inherits all properties of an R-Tree.
-Like an R-Tree, an EV-Tree is also depth-balanced.
-Leaf nodes are an array of leaf slots at the same level, each representing one rectangle.
-The value resides in leaf-nodes alone similar to B+ trees.
-Non-leaf nodes (or) internal nodes represent bounding box of each leaf-node or an internal node with a pointer pointing to the appropriate node.
-The root node contains at least two slots unless the root is a leaf node.
-In-addition to operations inherited from R-trees, EV-Trees support two additional operations, namely, splitting and trimming of rectangles to eliminate overlapping rectangles at leaves.
+R-Trees provide a reasonable way to represent both extent and epoch validity ranges in such a way as to limit the search space required to handle a read request.
+VOS provides a specialied R-Tree, caled an Extent-Validity tree (EV-Tree) to store and query versioned array indicies.
+In a traditional R-Tree implementation, rectangles are bounded and immutable.
+In VOS, the "rectangle" consists of the extent range on one axis and the epoch validity range on the other.
+However, the epoch validity range is unknown at the time of insert so all rectangles are inserted assuming an upper bound of infinity.
+Originally, the DAOS design called for splitting such in-tree rectangles on insert to bound the validity range but a few factors resulted in the decision to keep the original validity range.  First, updates to persistent memory are an order of magnitude more expensive than lookups.  Second, overwrites between snapshots can be deleted by aggregation thus maintaining a reasonably small history of overlapping writes.   As such, the EV-Tree implements a two part algorithm on fetch.
```

Comment:
a "two part algorithm" is mentioned, but then 4 steps are listed. Maybe group the 4 steps into the appropriate "two parts"